src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -248,8 +239,6 @@ public static void logAssertionStacktrace(AssertionError exception) {
       String html = VelocityWrapper.fillErrorLogRow(classList, exceptionMessage, Log.imageCounter);
       try {
         new Shooter().savePageScreenshot(Log.SCREEN_PATH + Log.imageCounter, driver);
-        CommonUtils.appendTextToFile(Log.SCREEN_PATH + Log.imageCounter + ".html",
-                                     Log.getPageSource(driver));
         CommonUtils.appendTextToFile(Log.LOG_PATH, html);
       } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

Reply:
Ditto :)